pkg/apis/componentconfig/cloudcore/v1alpha1/types.go

```diff
@@ -156,7 +156,7 @@ type CloudHub struct {
 // CloudHubQUIC indicates the quic server config
 type CloudHubQUIC struct {
 	// Enable indicates whether enable quic protocol
-	// default false
+	// default true
```

Comment:
I think it should not enable them all now, unless we have more tests and result. If we want to change the default value, plz open another PR :)